attributes/default.rb

```diff
@@ -1,5 +1,11 @@
 
 # Override cookbook attributes to install Java 7
-override['java']['openjdk_packages'] = [
-  "openjdk-7-jdk", "openjdk-7-jre-headless"
-]
+#override['java']['openjdk_packages'] = [
+#  "openjdk-7-jdk", "openjdk-7-jre-headless"
+#]
```

Comment:
I couldn't get Java 7 specifically to install and run without including this. And I'm pretty sure the Essentials plugin has a hard dependency on Java 7, which is why there are hacks (overridden attributes, symbolic links) to get it to work.